File: 4229-trim-trailing-vowels/trim-trailing-vowels.py
class Solution:
    def trimTrailingVowels(self, s: str) -> str:
        ind=-1
        for i in range(len(s)-1,-1,-1):
            if not (s[i]=="a" or s[i]=="e" or s[i]=="i" or s[i]=="o" or s[i]=="u"):
                ind=i
                break
        return s[0:ind+1]

File: 4229-trim-trailing-vowels/trim-trailing-vowels.java
class Solution {
    public String trimTrailingVowels(String s) {
        int ind=-1;
        for(int i=s.length()-1;i>=0;i--){
            char h=s.charAt(i);
            if(!(h=='a' || h=='e' || h=='i' || h=='o' || h=='u')){
                ind=i;
                break;
            }
        }
        return s.substring(0,ind+1);
    }
}